Framework/MDAlgorithms/src/IntegratePeaksMDHKL.cpp

```diff
@@ -0,0 +1,319 @@
+#include "MantidMDAlgorithms/IntegratePeaksMDHKL.h"
+#include "MantidDataObjects/MDFramesToSpecialCoordinateSystem.h"
+#include "MantidAPI/CommonBinsValidator.h"
+#include "MantidAPI/InstrumentValidator.h"
+#include "MantidAPI/WorkspaceUnitValidator.h"
+#include "MantidDataObjects/EventWorkspace.h"
+#include "MantidDataObjects/MDEventWorkspace.h"
+#include "MantidDataObjects/MDHistoWorkspace.h"
+#include "MantidGeometry/Instrument.h"
+#include "MantidKernel/CompositeValidator.h"
+
+#include "MantidGeometry/Crystal/IPeak.h"
+
+namespace Mantid {
+namespace MDAlgorithms {
+
+using Mantid::Kernel::Direction;
+// using Mantid::API::WorkspaceProperty;
+using namespace Mantid::DataObjects;
+using namespace Mantid::API;
+using namespace Mantid::Kernel;
+using namespace Mantid::Geometry;
+
+// Register the algorithm into the AlgorithmFactory
+DECLARE_ALGORITHM(IntegratePeaksMDHKL)
+
+//----------------------------------------------------------------------------------------------
+/**
+ * Constructor
+ */
+IntegratePeaksMDHKL::IntegratePeaksMDHKL() {}
```

Comment:
Is this any different than the default construct generated by the compiler? Would you remove it or use `= default;`?